app/renderer/js/main.ts

```diff
@@ -739,6 +739,14 @@ class ServerManagerView {
 		return !(url.endsWith('/login/') || this.tabs[tabIndex].webview.loading);
 	}
 
+	getActiveWebview(): any {
+		const selector = 'webview:not([class*=disabled])';
+		const webview = document.querySelector(selector);
+		if (webview) {
```

Comment:
Why the `if`?

Reply:
This is just to be sure that we get the webview. Can also remove this.